recipes/lora_dpo_distributed.py

```diff
@@ -620,6 +620,9 @@ def train(self) -> None:
                     policy_rejected_logits,
                 ) = self.concatenated_forward(self._model, batch)
 
+                policy_chosen_logits_mean = policy_chosen_logits.detach().mean()
+                policy_rejected_logits_mean = policy_rejected_logits.detach().mean()
+
```

Comment:
I think that the idea to do detach.mean was so we could delete the original ones and release memory. If that is true, are we missing the deletion part?

Reply:
good catch